templates/tour.html

```diff
@@ -105,18 +105,18 @@ <h3>Effortless network discovery</h3>
                     </a>
                 </div>
                 <h3>Simple device discovery</h3>
-                <p>MAAS can discover new <a href="https://docs.maas.io/en/installconfig-network-dev-discovery" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Interested in feature', 'eventAction' : 'Click', 'eventLabel' : 'Learn more about interfaces' });" class="p-link--external" aria-label="External link to device discovery page on MAAS documentation">devices</a> and <a href="https://docs.maas.io/en/installconfig-networking" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Interested in feature', 'eventAction' : 'Click', 'eventLabel' : 'Learn more about interfaces' });" class="p-link--external" aria-label="External link to install and config networking on MAAS documentation">network interfaces</a>:</p>
+                <p>MAAS can discover new <a href="https://maas.io/docs/network-discovery" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Interested in feature', 'eventAction' : 'Click', 'eventLabel' : 'Learn more about interfaces' });" class="p-link--external" aria-label="External link to device discovery page on MAAS documentation">devices and network interfaces</a>:</p>
                 <ul>
                     <li>Passively</li>
                     <li>On-demand</li>
                     <li>Periodically</li>
                 </ul>
                 <p>
-                    <a href="https://docs.maas.io/en/installconfig-network-dev-discovery"
+                    <a href="https://maas.io/docs/network-discovery"
                        onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Automation' });
                                 dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Interested in feature', 'eventAction' : 'Click', 'eventLabel' : 'Learn more about device discovery and subnet mapping' });"
                        class="p-link--external"
-                       aria-label="External link to MAAS release notes page on MAAS documentation">Learn more about device discovery and subnet mapping</a>
+                       aria-label="External link to the configuring networking page on MAAS documentation">Learn more about device discovery and subnet mapping</a>
```

Comment:
As these links are no longer external these `aria-label`s should be updated.

Reply:
Done